scripts/explain_manifest/fixtures/debian-10-amd64.txt

```diff
@@ -1,211 +0,0 @@
-- Path      : /usr/local/kong/include/google
```

Comment:
the two files are removed as they are not used at this time; debian packages are currently built on ubuntu actually